be/queue.h

```diff
@@ -0,0 +1,198 @@
+/* -*- C -*- */
+/*
+ * Copyright (c) 2015-2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
It's not that old. :-) Same in .c.

Reply:
`scripts/gen-src-skel` is broken and explanation of how to go forward hasn't been provided, so I'd prefer to copy-paste from existing files for now. When such information becomes available it would be easy to fix.